docs/reproducibility.md

```diff
@@ -0,0 +1,44 @@
+# Reproducibility
+
+The assumption here is that the original build was done using a Docker build. The
+goal is to reproduce the same MRENCLAVE value. 
+
+You need a couple pieces of information to begin:
+
+- The source MRENCLAVE value, get it from `https://ledgerurl/node/quotes/self`, e.g.:
```

Comment:
nit: just to make sure users would know they have to replace their ledger name in the given URL

```suggestion
- The source MRENCLAVE value, get it from `https://<LEDGER-URL>/node/quotes/self` (replace <LEDGER-URL> with the URL of your ledger), e.g.:
```